### PR TITLE
Modified to enable list of zookeeper hosts.

### PR DESCRIPTION
**Pull request by BratnerMichal**
In current state Proxima allows to define only single Zookeeper host even when Zookeeper supports list of hosts separated by comma. This modification should enable to define list of Zookeeper hosts.

### CODE REVIEW

**Review by je-ik — APPROVED** [review]
LGTM, thanks!

File: utils-zookeeper/src/main/java/cz/o2/proxima/utils/zookeeper/ZKGlobalWatermarkTracker.java [hunk 1]
```diff
@@ -152,15 +150,22 @@ private int getSessionTimeout(Map<String, Object> cfg) {
         .orElse(60000);
   }
 
-  private URI getZkUri(Map<String, Object> cfg) {
-    URI uri =
+  @VisibleForTesting
+  void parseZkUri(Map<String, Object> cfg) {
```
Comment (dmvk) [inline utils-zookeeper/src/main/java/cz/o2/proxima/utils/zookeeper/ZKGlobalWatermarkTracker.java:154]: I don't think we have to change the method contract here.

Can we just use `uri.getAuthority()` (authority represents host:port combinations) instead "manual" solutions?
```
    String.format("%s:%d", uri.getHost(), uri.getPort());
```
```
    zkConnectString = stringUri.substring(zkPrefix.length(), stringUri.indexOf(uri.getPath()));
```

**Review by dmvk — CHANGES_REQUESTED** [review]
This makes total sense 👍 Can you please simplify the code little bit by using `uri.getAuthority()` instead of manually constructing the zk connection string?

Thanks for the contribution 👍  🎉

File: utils-zookeeper/src/main/java/cz/o2/proxima/utils/zookeeper/ZKGlobalWatermarkTracker.java (hunk 1 above)
Comment (BratnerMichal) [reply utils-zookeeper/src/main/java/cz/o2/proxima/utils/zookeeper/ZKGlobalWatermarkTracker.java]: Good point, it leads to significantly smaller code modification.

**Review by dmvk — APPROVED** [review]
👍 perfect

**Pull request merged**